client/src/controllers/FormsetController.ts

```diff
@@ -148,20 +175,25 @@ export class FormsetController extends Controller<HTMLElement> {
     target.setAttribute(
       targetAttrName,
       (target.getAttribute(targetAttrName)?.split(' ') ?? [])
-        .filter((name) => name !== 'child')
+        // .filter((name) => name !== 'child')
```

Comment:
If we no longer need this, best to just remove the line instead of commenting out.